assets/css/wcsatt-write-panels.css

```diff
@@ -115,6 +115,10 @@
 	float: left;
 }
 
+#wcsatt_data .subscription_scheme_data p._satt_subscription_details span.wrap label {
```

Comment:
This seems to be targeting `label.wcs_hidden_label` which should be hidden in WCS 2.1/2.2.